src/lib/layouts/workflow-history-layout.svelte

```diff
@@ -164,3 +174,26 @@
     onClose={() => (showShortcuts = false)}
   />
 </div>
+
+<Modal
+  id="download-history"
+  large
+  bind:open={showDownloadPrompt}
+  confirmType="primary"
+  confirmText={translate('common.download')}
+  cancelText={translate('common.cancel')}
+  on:confirmModal={() => onDownloadClick()}
+  on:cancelModal={() => (showDownloadPrompt = false)}
+>
+  <h3 slot="title">
+    {translate('common.download-json')}
+  </h3>
+  <div slot="content" class="flex flex-col gap-4">
+    <ToggleSwitch
+      label={translate('events.decode-event-history')}
+      id="decode-event-history"
+      bind:checked={decodeEventHistory}
```

Comment:
Might be nice to default this to whatever the user has selected for `View decoded event history`

![Screenshot 2023-10-23 at 3 07 54 PM](https://github.com/temporalio/ui/assets/15069288/eea4d7a6-37d8-47ca-9bc4-552cc1510b76)

Reply:
👍 Made it a persistStore and used for both